project-base/storefront/components/Pages/Contact/contactFormMeta.tsx

```diff
@@ -31,23 +36,35 @@ export const useContactForm = (): [UseFormReturn<ContactFormType>, ContactFormTy
     return [formProviderMethods, defaultValues];
 };
 
+type PrivacyPolicyField = {
+    privacyPolicy: {
+        name: 'privacyPolicy';
+        label: JSX.Element;
+        errorMessage: string | undefined;
+    };
+};
+
+type InputFields = {
+    [key in keyof Omit<ContactFormType, 'privacyPolicy'>]: {
+        name: key;
+        label: string;
+        errorMessage: string | undefined;
+    };
+};
```

Comment:
It's not necessary (and would also deviate from what we have used to far) to have a separate type for just one field. I think it is good enough to just have 

```ts
 type InputFields = {
     [key in keyof ContactFormType]: {
         name: key;
         label: string | JSX.Element;
         errorMessage: string | undefined;
     };
 };
```

Reply:
Yes, that was my original thought as well, but then I got a type error in `textareaProps` label. However, I can adjust it just as it is in `textInputProps` to accept `ReactNode` instead of just plain `string`.